src/main/webapp/user_profile.jsp

```diff
@@ -209,9 +209,11 @@
                 </div>
                 <br/>
                 <input type="text" name="phone-input" id="edit-phone-number-input" required="true">
+                <input type="hidden" name="lat" id="lat-input">
+                <input type="hidden" name="lng" id="lng-input">
                 <br/>
                 <br/>
-                <input type="submit" />
+                <input type="submit" id="submit-button" title="You must mark your personal address on the map" disabled/>
```

Comment:
As discussed with @LeonardZ0731 , he adding a required red star next to the label on top of the map on his PR #88 to emphasize the fact that this is a required part of the form.